main.go

```diff
@@ -36,38 +34,30 @@ func main() {
 
 	ip = GetOutboundIP()
 	link := fmt.Sprintf("http://%v:%v", ip, port)
-	fmt.Printf("Hosting on %v (copied to clipboard!)\n", link)
-	err := clipboard.Init()
-	if err != nil {
-		panic(err)
-	}
-	clipboard.Write(clipboard.FmtText, []byte(link))
 
 	wd, err := os.Getwd()
 	if err != nil {
 		panic(err)
 	}
 
-	fmt.Printf("writing uploaded data to %s\n", wd+`\`+targetFolder)
```

Comment:
Please re-add the automatic folder opening, this is an intentional and important feature!